.github/workflows/jellyfish-tests.yml

```
@@ -14,16 +14,16 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - uses: actions/checkout@v3
         with:
           repository: JellyfishSDK/jellyfish
           ref: 'main'
 
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - uses: actions/checkout@v3
         with:
           path: defichain
 
-      - uses: actions/setup-node@46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea
+      - uses: actions/setup-node@v3
         with:
           node-version: '15'
```

```suggestion
          node-version: '16'
```
To match `jellyfishSDK` package.json engines